### DISCUSSION TITLE
Unable to apply for student dev pack (main branch not in the country I reside)

### DISCUSSION THREAD

**Discussion by Secret-Ambush** [community/community#60656] [GitHub Education]
### Hi everyone,

I'm currently enrolled as a student in a college that **doesn't have its main branch in the country where I reside**. Although the college does have a branch in my country, GitHub fails to recognize it. It keeps telling that the I am not in the same country as the college. 

The below is the message I keep getting:

_The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘Other’ option when submitting a GitHub Education support ticket._

_You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket._

I have filled in the application multiple times using both my **personal** and **college-issued** email id.
Using the college id, automatically selects the main branch as my college, while when using personal, I could enter details of the college branch I go to 

I tried with my **id card, fees receipt, fee advice for the new term, and my transcript** - as the academic proof all to no avail. 

I feel very discouraged, could someone please help me out with this?

**Comment by akkefa** [comment]
I am also facing the same issue and no one on the GitHub team taking notice of this issue.

https://github.com/orgs/community/discussions/60024
